app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java

```diff
@@ -48,76 +48,55 @@ public void initComponent(){
     public void onClick(View v){
         switch (v.getId()){
             case R.id.a_01:
```

Comment:
It can be improved by using one case and then you can pass id e.g; **a_01, a_02** etc from another function like you did it in function **intent** to pass values but doesn't matter this is just complex to read and change in future, so for now we can just leave this issue and approve your pull request